### DISCUSSION TITLE
Feature Request: Search to Add Font to Collection|收藏页面添加搜索功能

### DISCUSSION THREAD

**Discussion by Jiang10086** [character-map-uwp/Character-Map-UWP#316] [Ideas]
Thanks for developing this elegant and useful font tool! That's awesome.
I have a simple suggestion on the collection page in settings: now I have 578 font families installed on my computer, so it's difficult to find a certain font to add to collection, so I suggest to adding an search icon to add a certain font by search.
![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/67213568/ed40e4b4-36bc-49b1-a7fc-9d9a94152f9e)

**Comment by JohnnyWestlake** [comment]
Implemented in dev branch https://github.com/character-map-uwp/Character-Map-UWP/commit/7a9fe0cc4030aa5c6dbb528c67f4e24061ac54b5 - will update when it's merged into main and published. Thanks for the request!

![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/2319473/40ce4eba-8904-42a9-b651-d69cbc899505)

**Comment by JohnnyWestlake** [comment]
Should be available on the store in the next day or two

**Reply by Jiang10086** [reply]
Thanks, I have checked the lasted store version and found this feature, makes it much easier to manager fonts..
